driver/docker-container/driver.go

```diff
@@ -126,6 +134,38 @@ func (d *Driver) create(ctx context.Context, l progress.SubLogger) error {
 		if d.netMode != "" {
 			hc.NetworkMode = container.NetworkMode(d.netMode)
 		}
+		if d.memory != "" {
+			var memory uint64
+			memory, err := humanize.ParseBytes(d.memory)
+			if err != nil {
+				return err
+			}
+
+			hc.Resources.Memory = int64(memory)
+		}
```

Comment:
Matching with my recommendation above, I think we can do this:

```go
if d.memory != "" {
    var memory opts.MemBytes
    if err := memory.Set(d.memory); err != nil {
        return err
    }
    hc.Resources.Memory = int64(memory)
}
```

Reply:
done